go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/google/uuid v1.1.1 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/influxdata/line-protocol v0.0.0-20180522152040-32c6aa80de5e
-	github.com/influxdata/pkg-config v0.2.2
+	github.com/influxdata/pkg-config v0.2.3
```

Comment:
Why did we have to update `pkg-config` for this branch?

Reply:
I realized I was failing the tidy test and ran 'go mod tidy' which is the result of this. Not sure if I should be doing anything else for it.